.github/CODEOWNERS

```diff
@@ -952,6 +952,7 @@ x-pack/plugins/infra/server/lib/alerting @elastic/actionable-observability
 /x-pack/test/fleet_cypress @elastic/fleet
 /x-pack/test/fleet_functional @elastic/fleet
 /src/dev/build/tasks/bundle_fleet_packages.ts @elastic/fleet @elastic/kibana-operations
+/x-pack/plugins/fleet/server/services/elastic_agent_manifest.ts @elastic/fleet @elastic/obs-cloudnative-monitoring
```

Comment:
@elastic/kibana-operations - Could we grant this team write access to Kibana? I assume that's the issue here. They're writing to Kibana via CI anyway, so it feels proper for the team to have explicit write access.

Reply:
done